app/src/main/java/com/el/yello/presentation/onboarding/activity/EditNameActivity.kt

```diff
@@ -72,14 +72,15 @@ class EditNameActivity :
         val bundle: Bundle? = intent.extras
         intent.apply {
             if (bundle != null) {
-                viewModel.kakaoId = bundle.getLong(EXTRA_KAKAO_ID, 0).toString()
-                viewModel.nameText.value = bundle.getString(EXTRA_NAME, "")
-                viewModel.gender = bundle.getString(EXTRA_GENDER, "")
-                viewModel.email = bundle.getString(EXTRA_EMAIL, "")
-                viewModel.profileImg = bundle.getString(EXTRA_PROFILE_IMAGE, "")
+                with(viewModel) {
```

Comment:
```suggestion
                with(viewModel.bundle) {
```
묶는 김에 bundle까지 묶어줘도 될 것 같아요!